package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
     <name>nvidia_gmsl_driver_ros</name>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
```

Comment:
I would suggest 1.3.0, to be in-line with [current tags](https://github.com/UT-ADL/nvidia_gmsl_driver_ros/tags).

Reply:
done